less/footer.less

```diff
@@ -0,0 +1,70 @@
+.footer {
+  background-color: @gray-base;
+  font-size: @font-size-small;
+  line-height: @line-height-computed;
+
+  .footer-link,
```

Comment:
having their own class now, do we really need them to be nested? I suppose it's due to some specificity issue

Reply:
Yeap, that's the reason :)